dataprofiler/tests/profilers/test_graph_profiler.py

```diff
@@ -140,7 +140,7 @@ def setUpClass(cls):
                     "id": [
                         {},
                         {
-                            "bin_counts": "unchanged",
+                            "bin_counts": [[1], [1, 1, 2], []],
```

Comment:
updated outputs based on the changes made to `find_diff_of_lists_and_sets(stat1, stat2)`